src/client/clientmap.h

```diff
@@ -4,7 +4,6 @@
 
 #pragma once
 
-#include "irrlichttypes_bloated.h"
```

Comment:
Missing (provided through transitive includes, which we don't want)

```c++
#include "irr_aabb3d.h"
#include "irr_v2d.h"
#include "irr_v3d.h"
```

according to IWYU